src/PortingAssistant.Client.Analysis/IAnalysisHandler.cs

```diff
@@ -18,5 +18,7 @@ public interface IPortingAssistantAnalysisHandler
 
         Task<Dictionary<string, ProjectAnalysisResult>> AnalyzeSolutionIncremental(string solutionFilename, List<string> projects,
             string targetFramework = "netcoreapp3.1");
+
+        IAsyncEnumerable<ProjectAnalysisResult> AnalyzeSolutionGeneratorAsync(string solutionFilename, List<string> projects, string targetFramework = "netcoreapp3.1");
```

Comment:
Do we need an Init function to return a solutionanalysisresult that we can add these results to?

Reply:
we will build solutionanalysisresult object in ui proj